src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -0,0 +1,152 @@
+//#define ENERGY_PRESERVING_MONOCHROME
+
+float calcLightAttenuation( float lightDistance, float cutoffDistance, float decayExponent ) {
+
+	if ( decayExponent > 0.0 ) {
+
+	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+
+	}
+
+	return 1.0;
+
+}
+
+
+void BRDF_Lambert( const in IncidentLight incidentLight, const in GeometricContext geometryContext, const in vec3 diffuseColor, inout ReflectedLight reflectedLight ) {
+
+	float lambertianReflectance = saturate( dot( geometryContext.normal, incidentLight.direction ) );
+
+	#if defined( ENERGY_PRESERVING_MONOCHROME ) || defined( ENERGY_PRESERVING_RGB )
+
+		lambertianReflectance *= RECIPROCAL_PI;
```

Comment:
The equations that are implemented in three.js are detailed in [Background: Physics and Math of Shading](http://blog.selfshadow.com/publications/s2013-shading-course/hoffman/s2013_pbs_physics_math_notes.pdf). The equations of interest are equations (1) the reflectance equation, (4) the specular brdf, (7) the lambertian brdf, and (13) the reflectance equation for punctual light sources.

The equation you cite is the lambertian brdf = diffuseColor / pi. 

The reflectance equation has a factor of pi in front, which cancels out the pi in the denominator of the brdf. This is why we leave the factor of pi out, as noted in the original source code inline comments.

If there is a problem with the math, or a bug in our implementation of those equations, we should, of couse, fix it.